src/gfx/pal_spec.cpp

```diff
@@ -366,7 +365,7 @@ static void parseGPLFile(std::filebuf &file) {
 
 		++nbColors;
 		if (nbColors < maxNbColors) {
-			if (nbColors % options.nbColorsPerPal == 0) {
+			if (nbColors % options.nbColorsPerPal == 1) {
```

Comment:
I had to use `gdb` to find this bug... `Segmentation fault (core dumped)` is such a useless error message.

Reply:
Though to be fair, you need specialised software anyway to debug segfaults.

Reply:
Or a specialized language where `arr[i] = x;`  is bounds-checked... like ALGOL. :P